Crashlytics/Crashlytics/Public/FirebaseCrashlytics/FIRCrashlytics.h

```diff
@@ -82,7 +82,7 @@ NS_SWIFT_NAME(Crashlytics)
  * @param value The value to be associated with the key
  * @param key A unique key
  */
-- (void)setCustomValue:(id)value forKey:(NSString *)key;
+- (void)setCustomValue:(nullable id)value forKey:(NSString *)key;
```

Comment:
This is potentially a breaking change for Swift users, so may need to wait until Firebase 9. cc: @ryanwilson

Reply:
Thanks for tagging! As far as I'm aware going from `nonnull` -> `nullable` isn't a breaking change as the source can stay the same, but the other way around can cause a compilation failure since the optional isn't unwrapped. 

Since this fixes a bug, I think it's reasonable to land before then. Happy to take the blame if it does break though :)

Reply:
In my local testing it didn't seem to be an issue, but I'll keep an eye out!